kserve/README.md

```diff
@@ -21,6 +21,32 @@ bash scripts/setup.sh
 
 > *Note: It may be necessary to run with `root` privileges if the default `/raid/nim` is not accessible.
 
+`NOTE:` If you have StorageClass and plan to use that please follow the below steps, please make sure run the below steps before run `bash scripts/setup.sh`
+
+  Run the below command to back up the `nvidia-nim-cache.yaml`
+  ```
+  mv scripts/nvidia-nim-cache.yaml scrips/nvidia-nim-cache-backup.yaml
+  ```
+  create a new file with PersistenceVolumeClaim with below command and update the storageClassName accordingly
+  
+  `Note:` Storage Class should have an accessModes option as `ReadWriteMany`
+
+   ```
+   cat <<EOF | tee scripts/nvidia-nim-cache.yaml
+   apiVersion: v1
+   kind: PersistentVolumeClaim
+   metadata:
+     name: nvidia-nim-pvc
+   spec:
+     accessModes:
+       - ReadWriteMany
+     storageClassName: <storage-class-name>
```

Comment:
People are going to copy/paste this and it won't work.

Reply:
@supertetelman it's hard to predict or set the `StorageClassName` as a variable and update. that's where I have mentioned the statement above with update the storage class name. 

I have added additional line to find the storageClass as well